src/frequenz/sdk/timeseries/battery_pool/battery_pool.py

```diff
@@ -84,15 +90,144 @@ def __init__(  # pylint: disable=too-many-arguments
             )
 
         self._min_update_interval = min_update_interval
+
+        self._power_distributing_sender = power_distributing_sender
         self._active_methods: dict[str, MetricAggregator[Any]] = {}
 
         self._namespace: str = f"battery-pool-{self._batteries}-{uuid.uuid4()}"
+        self._power_distributing_namespace: str = f"power-distributor-{self._namespace}"
+        self._channel_registry: ChannelRegistry = channel_registry
         self._formula_pool: FormulaEnginePool = FormulaEnginePool(
             self._namespace,
             channel_registry,
             resampler_subscription_sender,
         )
 
+    async def set_power(
+        self,
+        power: float,
+        *,
+        adjust_power: bool = True,
+        request_timeout: timedelta = timedelta(seconds=5.0),
+        include_broken_batteries: bool = False,
+    ) -> None:
+        """Set the given power for the batteries in the pool.
+
+        Power values need to follow the Passive Sign Convention (PSC). That is, positive
+        values indicate charge power and negative values indicate discharge power.
+
+        When not using the Passive Sign Convention, the `charge` and `discharge` methods
+        might be more convenient.
+
+        Args:
+            power: The power to set for the batteries in the pool.
+            adjust_power: If True, the power will be adjusted to fit the power bounds,
+                if necessary. If False, then power requests outside the bounds will be
+                rejected.
```

Comment:
maybe we should have a better name for `adjust_power`, for example `apply_power_bounds`, because the former does not tell me what this parameter is doing, in fact, I was scratching my face and wondering the whole purpose of this function just by looking at that boolean in that context. :-)

With a better name, hopefully (like proposed), it should be more clear what impact this boolean parameter will have.

Same applies to the other `adjust_power` occurrences below.

Reply:
Another alternative could be `adjust_to_bounds`. at least for me it is not super clear what applying a bound means.

Reply:
I think it is impossible to encode everything the flag does into its name,  we will have to read the docs,  because it is not trivial and it needs info about
1. when/why is it required - when requests exceed the bounds
2. what does it do when true - adjust power to fit within the bounds and forward request
3. what does it do when false - reject request

And `apply_power_bounds` always happens.  What to do after that is the question - adjust the power value to fit the bounds, or reject the request.

Reply:
But `adjust_to_bounds` is nicer.  If there's code that says:

```python
await battery_pool.charge(2000, adjust_to_bounds=True)
```
maybe it is obvious to first time users without reading the docs.  Hopefully,  in most scenarios they want out default value (True) anyway and won't have to deal with it.

Reply:
> I think it is impossible to encode everything the flag does into its name, we will have to read the docs, because it is not trivial and it needs info about

Yeah, I was about to say the same thing, but in this case I also felt that there was room for improvement :)

We need to strike a balance. And like `sop`, if something is super hard to make clear in a *indetifier*, I think it is better to have it obscure to force people to look at the docs instead of having something more meaningful but misleading.